packages/itwinui-css/src/utils/sizes.scss

```diff
@@ -1,7 +1,6 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
-@import '../text/mixins';
 
 @mixin iui-avatar-size($size: $iui-icons-xl, $font-size: $iui-font-size-leading) {
```

Comment:
Maybe rename file to `avatar-sizes`. Similar to `icon-sizes` in styles folder.

Reply:
Renamed